src/components/Common/FullWidthButton.tsx

```diff
@@ -1,28 +1,38 @@
 import classNames from 'classnames'
 import { Button } from 'decentraland-ui/dist/components/Button/Button'
+import { Loader } from 'decentraland-ui/dist/components/Loader/Loader'
 
 import './FullWidthButton.css'
 
 interface Props {
-  onClick?: (param?: unknown) => void
+  onClick?: () => void
   children: React.ReactNode
   className?: string
-  link?: string
+  href?: string
   newWindow?: boolean
+  loading?: boolean
 }
 
-const FullWidthButton = ({ onClick, children, className, link, newWindow = false }: Props) => {
+// TODO: FullWidthButton should render Link when href is provided
```

Comment:
I think you can do:

```
<Button
  as={href ? Link : undefined}
  ...
/>
```

Reply:
Definitely! I'll change it in another PR.